app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] From the project requirement, Your post should have a method that updates the posts counter for a user. So I suggest you create a `update_post_counter` private method to update the post counter for a user after a post is created. :relaxed: